pkg/api/v1/conversion.go

```diff
@@ -164,7 +219,7 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Endpoints",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "metadata.name":
+			case "metadata.name", "metadata.namespace":
```

Comment:
Could you make this a loop for different kinds? and also in extensions/v1beta1/conversion.go